test/ig/IGExporter.IG.test.ts

```diff
@@ -1683,11 +1707,6 @@ describe('IGExporter', () => {
     });
 
     it('should not warn on deeply nested resources when implicated by the path-resource parameter', () => {
-      defs = new FHIRDefinitions();
-      loadFromPath(path.join(__dirname, '..', 'testhelpers', 'testdefs'), 'r4-definitions', defs);
-      fixtures = path.join(__dirname, 'fixtures', 'customized-ig-with-nested-resources');
-      loadCustomResources(path.join(fixtures, 'input'), fixtures, config.parameters, defs);
-
```

Comment:
Is there a reason these lines were deleted? I think we still want them to set this test up correctly.

Reply:
No, this was redundant code (it copied from setup that was established previously).

Reply:
Hm it looks like the last line is actually different and the `config.parameters` is passed in to `loadCustomResources`, and based on the test description, it looks like that might be what this is trying to test. But if that's the case, I'm not really sure why the test is still passing. Maybe @julianxcarter knows since it looks like he added this test. But if not, it's probably fine to keep these lines removed.

Reply:
After looking at this test a little more, I do think we want to keep the lines that were removed. We want to be testing that passing in the config.parameters does properly add the nested resources without providing a warning. The problem is that the test here isn't exactly working as expected. We will need to include these four lines again, but we also need to redefine the `exporter` so that it has the `defs` that are loaded with the new parameter in `loadCustomResources`. We also probably only want `config.parameter` to be set in this test, so we probably will want to move where that happens from in the `beforeAll` into this test. That means we'd also need to redefine `pkg` to create a package with the appropriate config.

We can talk about if these are changes we want to make as part of this PR or a separate task when you're back!

Reply:
Sound good, undeleted the changes for now. Took a look at it again and since we want `config.parameter` set it makes sense that it's not dine in the `beforeAll`.